### PR TITLE
add connector doujinku

### LINKED ISSUE #6520: doujinku.xyz
**Name of the website**
doujinku.xyz
https://doujinku.xyz/

Website urls (examples below)

Site: https://doujinku.xyz/
Manga list: https://doujinku.xyz/manga/list-mode/
Manga example: https://doujinku.xyzdoujinku.xyz/manga/the-pleasure-shop/
Chapter online viewer example: https://doujinku.xyz/the-pleasure-shop-chapter-62/

**Languages**
List of languages supported by the website
Indonesian

**Website relationship*
If applicable describe the relation with any other website (eg : "alternative domain, copy of ...").

**Additional details**
Add any other context details that you may have found (like template to reuse)

### PR DESCRIPTION
**Pull request by akn19**
fixes: https://github.com/manga-download/hakuneko/issues/6520

### CODE REVIEW

File: src/web/mjs/connectors/Doujinku.mjs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+
+export default class Doujinku extends WordPressMangastream {
+
+    constructor() {
+        super();
+        super.id = 'doujinku';
+        super.label = 'Doujinku';
+        this.tags = ['hentai', 'indonesian'];
```
Comment (MikeZeDev) [inline src/web/mjs/connectors/Doujinku.mjs]: One may want to add 'manga' or maybe 'manhwa' tag.

**Review by MikeZeDev — APPROVED** [review]
Minor change required. Thanks for your contribution.

**Review by MikeZeDev — COMMENTED** [review]
icon is not a png but a webp.  Pleace change it to a 64x64 px PNG.

File: src/web/mjs/connectors/Doujinku.mjs (hunk 1 above)
Comment (andrizan) [reply src/web/mjs/connectors/Doujinku.mjs]: > One may want to add 'manga' or maybe 'manhwa' tag.

done

**Comment by akn19** [comment]
> icon is not a png but a webp. Pleace change it to a 64x64 px PNG.

done

**Pull request merged**